apps/src/templates/progress/ProgressDetailToggle.story.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ProgressDetailToggle from './ProgressDetailToggle';
+import { Provider } from 'react-redux';
+import {combineReducers, createStore} from 'redux';
+import progress, { setIsSummaryView } from '@cdo/apps/code-studio/progressRedux';
+
+export default storybook => {
+  const storyTable = [];
+
+  // Stick these in anonymous functions so that each instance gets its own
+  // store
+  (function () {
+    const store = createStore(combineReducers({progress}));
+    storyTable.push({
+      name:'isSummary is true',
+      story: () => (
+        <Provider store={store}>
+          <ProgressDetailToggle/>
+        </Provider>
+      )
+    });
+  })();
+
+
+  (function () {
+    const store = createStore(combineReducers({progress}));
+    store.dispatch(setIsSummaryView(false));
+    storyTable.push({
+      name:'isSummary is false',
+      story: () => (
+        <Provider store={store}>
+          <ProgressDetailToggle/>
+        </Provider>
+      )
+    });
+  })();
+
+  storybook
+    .storiesOf('ProgressDetailToggle', module)
+    .addStoryTable(storyTable);
+};
```

Comment:
I suspect that using redux inside storybook might be a common enough use case that it might be worth directly supporting it in `addStoryTable` or perhaps a differently named addon/decorator.

An API like the following would be nice:

```jsx
storybook
  .storiesOf('ProgressDetailToggle', module)
  .withReduxStore(() => createStore(combineReducers({progress})))
  .addStoryTable([
    {
      name: 'isSummary is false',
      dispatch: [setIsSummaryView(false)],
      story: store => <ProgressDetailToggle store={store} />
    },
    ...
  ]);
```